src/joystick/hidapi/SDL_hidapi_flydigi.c

```diff
@@ -323,7 +323,7 @@ static void HIDAPI_DriverFlydigi_HandleStatePacket(SDL_Joystick *joystick, SDL_D
 {
     Sint16 axis;
     Uint64 timestamp = SDL_GetTicksNS();
-    if (data[0] != 0x04 && data[0] != 0xFE) {
+    if (data[0] != 0x04 && data[1] != 0xFE) {
```

Comment:
Assuming the intention is correct (@slouken ), does `&&` need changing to `||` ??

Reply:
Yep, I'll take care of it.